pypit/spectrographs/spectro_utils.py

```diff
@@ -24,13 +27,28 @@ def load_spec_class(spectrograph=None, data_file=None):
         spec_class: Spectrograph class (or one of its children)
 
     """
+    spec_class = None
     if spectrograph is not None:
-        if 'keck_lris' in spectrograph:
-            spec_class = lris.LRISSpectrograph()
+        if 'keck_lris_blue' in spectrograph:
+            spec_class = keck_lris.KeckLRISBSpectrograph()
+        elif 'keck_lris_red' in spectrograph:
+            spec_class = keck_lris.KeckLRISRSpectrograph()
         elif 'keck_deimos' in spectrograph:
-            spec_class = deimos.DEIMOSSpectrograph()
+            spec_class = keck_deimos.KeckDEIMOSSpectrograph()
+        elif 'keck_nirspec' in spectrograph:
+            spec_class = keck_nirspec.KeckNIRSPECSpectrograph()
+        elif 'shane_kast_blue' in spectrograph:
+            spec_class = shane_kast.ShaneKastBlueSpectrograph()
+        elif 'shane_kast_red' in spectrograph:
```

Comment:
Maybe add a TODO (if you agree) - perhaps we should only have one kast red spectrograph, and determine which detector it is based on the date in the header? This would avoid the user needing to know what instrument to feed in.

Eventually, we might consider just having kast, and then split red and blue also in the spectro_class... this last suggestion maybe a little too out there...  ;-)

Reply:
Yes you see... it is all in the headers;)